src/ServiceV2/Mailer/Email/DuplicatedUsernameEmail.php

```diff
@@ -2,20 +2,28 @@
 
 namespace App\ServiceV2\Mailer\Email;
 
+use App\Entity\Centre;
+use App\Entity\Region;
 use App\Entity\User;
+use Symfony\Bridge\Twig\Mime\TemplatedEmail;
 use Symfony\Component\Mime\Email;
 
 readonly class DuplicatedUsernameEmail
```

Comment:
Why didn't you extends `LocalizedTemplatedEmail` ?

Reply:
I don't think we ever gonna translate it ?

Reply:
Translation is not mandatory, it could be helpfull to lighten this email class